docs/report/report.tex

```diff
@@ -44,39 +44,69 @@ \section{Einleitung}
 
 \subsection{Unternehmen}
 \begin{quote}
-  \textit{Ich denke, dass jede allgemein nützliche Information frei sein sollte. Mit 'frei' beziehe ich mich nicht auf den Preis, sondern auf die Freiheit, Informationen zu kopieren und für die eigenen Zwecke anpassen zu können. Wenn Informationen allgemein nützlich sind, wird die Menschheit durch ihre Verbreitung reicher, ganz egal, wer sie weiter gibt und wer sie erhält.}
+  \textit{Ich denke, dass jede allgemein nützliche Information frei sein sollte.
+  Mit 'frei' beziehe ich mich nicht auf den Preis, sondern auf die Freiheit, Informationen zu kopieren und für die eigenen Zwecke anpassen zu können.
+  Wenn Informationen allgemein nützlich sind, wird die Menschheit durch ihre Verbreitung reicher, ganz egal, wer sie weiter gibt und wer sie erhält.}
```

Comment:
Das ist doch ein Zitat. Da würde ich glaube ich nichts drin ändern. Es sei denn Julian hat falsch Zitiert

Reply:
hab dort nur im der tex zeilenumbrüche reingepackt, um das in der tex besser lesbarer zu haben

Reply:
Ahh ok :D